modules/local/sum_taxonomy.nf

```diff
@@ -31,7 +31,7 @@ process SUM_TAXONOMY {
     library(readr)
     library(tidyr)
     library(stringr)
-    library(purrr)
+    library(tidyverse)
```

Comment:
I don't think this is the best way to solve the problem. All the libraries above this are included in Tidyverse, so none of them needs to be included if you have this line. I guess there's a specific library that's missing. What's the error message you get?